packages/vscode-ext/package.json

```diff
@@ -142,6 +144,13 @@
     "eslint": "^8.18.0",
     "glob": "^8.0.3",
     "js-yaml": "^4.1.0",
-    "typescript": "^4.7.4"
+    "typescript": "^4.7.4",
+    "vsce-yarn-patch": "^1.66.2"
```

Comment:
This is for our monorepo compatibility (vsce is a tool for packaging vscode extensions), see https://github.com/microsoft/vscode-vsce/issues/300#issuecomment-978256849.

Yes, it's a fork. Which haven't been updated in 2 years. Seems like everyone with a monorepo uses it. Sigh.